TurkishFilter/sections/general_extensions.txt

```diff
@@ -73,7 +73,7 @@ dizipia.com#%#//scriptlet('abort-on-property-write', 'popURL')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/161599
 erosfilmizle.*#%#//scriptlet('prevent-window-open')
 ! https://github.com/AdguardTeam/AdguardFilters/issues/161110
-hdfilmizle.org#%#//scriptlet("set-cookie", "popundr", "1")
+hdfilmizle.org,dizifast.net#%#//scriptlet("set-cookie", "popundr", "1")
```

Comment:
Not an error, but we prefer to not use `set-cookie` if the problem can be fixed without writing cookies. The rule is ok by itself.
This script is popular and can be prevented by this rule for `puShown`
https://github.com/AdguardTeam/AdguardFilters/blob/0639f17d9bbe672ce0ad5d832d74efb5d32152a1/BaseFilter/sections/general_extensions.txt#L108

Reply:
Danm, I posted the wrong link.
Please apply this change to the rule in Turkish filter
https://github.com/AdguardTeam/AdguardFilters/blob/3237a6520f543ee8aeca9e1a7713a696fb05e42b/TurkishFilter/sections/general_extensions.txt#L15